File: gui/workspace/apps/reactive-app/app/model/Person.js
Ext.define('ReactiveApp.model.Person', {
	extend: 'Ext.data.Model',

	fields: [
	
		{
			type: 'int',
			name: 'id'
		},
		{
			type: 'string',
			name: 'firstName'
		},
		{
			type: 'string',
			name: 'lastName'
		},
		{
			type: 'string',
			name: 'company'
		}
	]

});

File: server/spring-reactive-r2dbc/src/main/java/io/naztech/r2dbc/model/Person.java
package io.naztech.r2dbc.model;

import org.springframework.data.annotation.Id;
import org.springframework.data.relational.core.mapping.Table;

@Table("person")
public class Person {

	@Id
	private Long id;

	private String firstName;

	private String lastName;

	private String company;

	public Long getId() {
		return id;
	}

	public void setId(Long id) {
		this.id = id;
	}

	public String getFirstName() {
		return firstName;
	}

	public void setFirstName(String firstName) {
		this.firstName = firstName;
	}

	public String getLastName() {
		return lastName;
	}

	public void setLastName(String lastName) {
		this.lastName = lastName;
	}

	public String getCompany() {
		return company;
	}

	public void setCompany(String company) {
		this.company = company;
	}
}